include/umf/memory_pool.h

```diff
@@ -42,6 +42,24 @@ umf_result_t umfPoolCreate(const umf_memory_pool_ops_t *ops,
                            umf_memory_provider_handle_t provider, void *params,
                            umf_memory_pool_handle_t *hPool);
 
+///
+/// @brief Creates new memory pool with new memory provider.
+///        Memory provider is created from specified provider_ops and provider_params
+///        and passed to pool_ops::initialize. The provider is owned by the pool and
+///        destroyed in umfPoolDestroy.
+/// @param pool_ops instance of umf_memory_pool_ops_t
+/// @param pool_params pointer to pool-specific parameters
+/// @param provider_ops instance of umf_memory_pool_ops_t
```

Comment:
`umf_memory_provider_ops_t` instead of `umf_memory_pool_ops_t`

Reply:
Done.